packages/playground/blueprints/src/lib/steps/login.spec.ts

```diff
@@ -24,7 +24,7 @@ describe('Blueprint step installPlugin', () => {
 	it('should log the user in', async () => {
 		await login(php, {});
 		const response = await requestHandler.request({
-			url: '/wp-admin',
+			url: '/wp-admin/',
```

Comment:
This fixes the test because the handler now returns a redirect when the request URI identifies an existing dir without including a trailing slash.